addon/-private/system/model/internal-model.js

```diff
@@ -849,4 +851,21 @@ InternalModel.prototype = {
 
     return reference;
   }
-};
+}
+
+if (isEnabled('ds-reset-attribute')) {
+  /*
+     Returns the latest truth for an attribute - the canonical value, or the
+     in-flight value.
+
+     @method latestTrueValue
+     @private
+  */
+  InternalModel.prototype.latestTrueValue = function latestTrueValue(key) {
```

Comment:
Though this is an internal method, I am not not too happy about the name here to be honest. Coming up with a better one is difficult though 😔  What do you think of `lastPreassignedAttributeValue()` or even `lastCanonicalOrInFlightAttributeValue()` or something? I know this is a tiny nitpick, but I was hoping we find a better name here...